packages/utils/src/lib/format-command-log.integration.test.ts

```diff
@@ -0,0 +1,54 @@
+import path from 'node:path';
+import { describe, expect, it } from 'vitest';
+import { removeColorCodes } from '@code-pushup/test-utils';
+import { formatCommandLog } from './format-command-log.js';
+
+describe('formatCommandLog', () => {
+  it('should format simple command', () => {
+    const result = removeColorCodes(
+      formatCommandLog('npx', ['command', '--verbose']),
+    );
+
+    expect(result).toBe('$ npx command --verbose');
+  });
+
+  it('should format simple command with explicit process.cwd()', () => {
+    const result = removeColorCodes(
+      formatCommandLog('npx', ['command', '--verbose'], process.cwd()),
+    );
+
+    expect(result).toBe('$ npx command --verbose');
+  });
+
+  it('should format simple command with relative cwd', () => {
+    const result = removeColorCodes(
+      formatCommandLog('npx', ['command', '--verbose'], './wololo'),
+    );
+
+    expect(result).toBe(`wololo $ npx command --verbose`);
+  });
+
+  it('should format simple command with relative cwd in parent folder', () => {
+    const result = removeColorCodes(
+      formatCommandLog('npx', ['command', '--verbose'], '..'),
+    );
+
+    expect(result).toBe(
+      `${path.relative(process.cwd(), '..')} $ npx command --verbose`,
+    );
+  });
+
+  it('should format simple command with absolute cwd', () => {
+    const result = removeColorCodes(
+      formatCommandLog(
+        'npx',
+        ['command', '--verbose'],
+        '/tmp/path/to/somewhere',
+      ),
+    );
+
+    expect(result).toBe(
+      `${path.relative(process.cwd(), '/tmp/path/to/somewhere')} $ npx command --verbose`,
+    );
+  });
```

Comment:
It's not very good practice to use expressions instead of literals in test assertions. Especially when the assertion reuses internal implementation, the tests end up being a self-fulfilling prophecy, instead of describing behaviour from the outside. Yes, "the function that returns `path.relative(...)` does indeed match `path.relative(...)`", obviously, but what does that mean? What _value_ does it actually return?

I think both of these tests could be merged into one, with a literal assertion:

```suggestion
  it('should format simple command using relative path to parent directory', () => {
    const result = removeColorCodes(
      formatCommandLog(
        'npx',
        ['command', '--verbose'],
        path.dirname(process.cwd())),
    );

    expect(result).toBe('.. $ npx command --verbose');
  });
```